concordium-sdk/src/test/java/com/concordium/sdk/walletexports/browserWallet.json

```
@@ -0,0 +1,25 @@
+{
+  "type": "concordium-browser-wallet-account",
+  "v": 0,
+  "environment": "testnet",
+  "value": {
+    "accountKeys": {
+      "keys": {
+        "0": {
+          "keys": {
+            "0": {
+              "signKey": "56f60de843790c308dac2d59a5eec9f6b1649513f827e5a13d7038accfe31784",
+              "verifyKey": "2c1c9c13c93c460ab723c69c0c39f0aa3fc1eda9dbbb4feff85dce222ba67754"
+            }
+          },
```

```suggestion
        "17": {
          "keys": {
            "32": {
              "signKey": "56f60de843790c308dac2d59a5eec9f6b1649513f827e5a13d7038accfe31784",
              "verifyKey": "2c1c9c13c93c460ab723c69c0c39f0aa3fc1eda9dbbb4feff85dce222ba67754"
            }
          },
```

This should also work.